.github/PULL_REQUEST_TEMPLATE/bug-fix.md

```diff
@@ -0,0 +1,78 @@
+---
+name: Bug fix
+about: Use this if your PR fixes a bug in EssentialsX.
+labels: 'type: bugfix'
+---
+
+<!-- EssentialsX bug fix submission guide
+
+NOTE: Failure to fill out this template properly may result in your PR being
+      delayed or ignored without warning.
+
+Don't put anything inside this block, as it will be hidden when you post your
+PR. Make sure to also not type between any arrows in the template, as this text
+will also be hidden.
+
+Want to discuss your PR before submitting it? Join the EssentialsX Development
+server: https://discord.gg/CUN7qVb
+
+If you are submitting a bug fix, please follow the following steps:
+
+1.  Fill out the template in full.
+      This includes providing screenshots and a link to the original bug 
+      report. If there isn't an existing bug report, we recommend opening a new
+      detailed bug report BEFORE opening your PR to fix it, else your PR may be
+      delayed or rejected without warning.
+      
+      You can open a new bug report by following this link:
+      https://github.com/EssentialsX/Essentials/issues/new/choose 
+
+2.  When linking logs or config files, do not attach them to the post!
+      Copy and paste any logs into https://gist.github.com/, then paste a
+      link to them in the relevant parts of the template. Do not use Hastebin
+      or Pastebin, as this can cause issues with future reviews.
+      DO NOT drag logs into this box!
+
+3.  If you are fixing a performance issue, please include a link to a
+    Timings and/or profiler report, both before and after your PR.
+
+4.  If you are fixing a visual bug, such as in commands, please include
+    screenshots so that we can more easily review the proposed fix.
+    (You may drag screenshots directly into this box.)
+
+-->
+
+### Information
+
+<!--
+    Replace #nnnn with the number of the original issue. If this PR fixes
+    multiple issues, you should repeat the "fixes #nnnn" for each issue. 
+-->
+
+This PR fixes #nnnn. 
+
+### Details
+
+**Proposed fix**  
+<!-- Type a description of your proposed fix below this line. -->
+
+**Environments tested**  
+<!--
+    Below this line, put an "x" inside the box for the environments you have
+    tested this bug fix on, and if relevant alter the OS and Java version
+    accordingly. If this feature does not apply to an environment, strike
+    through the environment using ~~strikethrough~~. If you have tested on
+    other environments, add a new line with relevant details.
+-->
+
+- [ ] Paper 1.16.3 (any OS, any Java 8+ version)
+- [ ] CraftBukkit/Spigot/Paper 1.12.2 (any OS, any Java 8+ version)
+- [ ] CraftBukkit 1.8.8 (any OS, any Java 8+ version)
+
+**Demonstration**  
+<!--
+    Below this block, include screenshots/log snippets from before and after as
+    necessary. If you have created or used a test case plugin, please link to
+    a download of the plugin, source code and exact version used where
```

Comment:
Oxford comma after source code, argue accordingly. I'm always for adding so I recommend it here.